bip-0141.mediawiki

```diff
@@ -57,21 +57,17 @@ If there are more than one scriptPubKey matching the pattern, the one with highe
 
 A scriptPubKey (or redeemScript as defined in BIP16/P2SH) that consists of a 1-byte push opcode (for 0 to 16) followed by a data push between 2 and 32 bytes gets a new special meaning. The value of the first push is called the "version byte". The following byte vector pushed is called the "witness program".
 
-Witness verification logic is either
-# triggered by a scriptPubKey that is exactly a push of a version byte, plus a push of a witness program, or
-# triggered when a scriptPubKey is a P2SH script, and the BIP16 redeemScript pushed in the scriptSig is exactly a push of a version byte plus a push of a witness program.
+There are two cases in which witness validation logic are triggered. Each case determines the location of the witness byte and program, as well as the form of the scriptSig:
```

Comment:
please keep using "version byte" since it is used everywhere (not just in BIP141)